include/seqan3/core/algorithm/detail/all.hpp

```diff
@@ -13,4 +13,5 @@
 #pragma once
 
 #include <seqan3/core/algorithm/detail/algorithm_executor_blocking.hpp>
+#include <seqan3/core/algorithm/detail/execution_handler_parallel.hpp>
```

Comment:
I think in general we don't want detail to be included inside of the all.hpp.
It basically is no functionality for the user and should be included through the public headers.

Reply:
So we leave it for now but I would remove it later. Lets discuss this today in the group meeting